packages/amplify-graphql-api-construct/src/types.ts

```diff
@@ -752,6 +752,12 @@ export interface AmplifyGraphqlApiProps {
    * For more information, refer to https://docs.amplify.aws/lib/datastore/getting-started/q/platform/js/
    */
   readonly dataStoreConfiguration?: DataStoreConfiguration;
+
+  // TODO: decide final naming before merging to main
+  /**
+   * The table map for the imported tables. The key is the model type name defined in schema; the value is the table name of the existing table
```

Comment:
1. This can happen in a followup PR, but let's make sure to highlight some of the major constraints. Off the top of my head: 1/ the existing table must have been created by Amplify Gen1, 2/ the existing table must be in the same account and region as the API.
2. For parallelism with the existing `stackMappings` input property, we could consider a name like `importedAmplifyDynamoDBTableMapping`. I'm not 100% sold on that though, so it's just a suggestion to consider for naming review.
3. While we're in this file (again can happen in a followup PR), can you correct the typo in the comment on L746? `If no outputStorageStrategey` -> `If no outputStorageStrategy`